pkg/ipmi/ipmi_dev.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
The general guidance on names like this is
pkg/ipmi/dev.go
as the name pkg/ipmi/ipmi_dev.go "stutters", i.e. repeats a name it need not repeat. I'd suggest changing this to dev.